purchases-sample/src/main/res/layout/fragment_overview.xml

```diff
@@ -9,6 +9,8 @@
         <variable
                 name="purchaserInfo"
                 type="com.revenuecat.purchases.PurchaserInfo" />
+
+        <import type="android.view.View" />
```

Comment:
what's this for?

Reply:
that's so i can access View.VISIBLE when setting visibility on the manage button

Reply:
Oh I see